tgbotapi.api/src/commonMain/kotlin/dev/inmo/tgbotapi/extensions/api/edit/Edits.kt

```diff
@@ -66,13 +66,13 @@ suspend fun TelegramBot.edit(
     proximityAlertRadius: Meters? = null,
     businessConnectionId: BusinessConnectionId? = chatId.businessConnectionId,
     replyMarkup: InlineKeyboardMarkup? = null
-) = editLiveLocation(chatId, messageId, latitude, longitude, livePeriod, horizontalAccuracy, heading, proximityAlertRadius, businessConnectionId, replyMarkup)
+): ContentMessage<LocationContent> = editLiveLocation(chatId, messageId, latitude, longitude, livePeriod, horizontalAccuracy, heading, proximityAlertRadius, businessConnectionId, replyMarkup)
```

Comment:
Are you sure that here `ContentMessage<LocationContent>` and not `ContentMessage<LiveLocationContent>`?

Reply:
this causes errors because `editLiveLocation` returns `ContentMessage<LocationContent>`, which in turn happens because `EditChatMessageLiveLocation` request ascends `EditChatMessage<LocationContent>`

thus, all that time `EditChatMessageLiveLocation` returned a location that was not live

Reply:
Could you please check if it is right? It must returns `LiveLocationContent` as a content in case of success, if I correctly remember

Reply:
yes, it should, but it does not because of this:
```kotlin
data class EditChatMessageLiveLocation(
    ...
) : /* here! */ EditChatMessage<LocationContent>, EditReplyMessage, EditLocationMessage {
...
}
```